webapp/channels/src/components/channel_move_to_sub_menu_old/index.tsx

```diff
@@ -41,7 +41,7 @@ type Props = {
     inHeaderDropdown?: boolean;
 };
 
-const ChannelMoveToSubMenu = (props: Props) => {
+const ChannelMoveToSubMenuOld = (props: Props) => {
```

Comment:
I think it was called "old" because we had two versions of the component for the LHS which uses MUI and the channel header which doesn't. I can't remember if it was because of this specific bug though or if it was just because rendering an MUI menu inside of a React Bootstrap menu caused other problems